arwen/host/execution_test.go

```diff
@@ -772,8 +797,16 @@ func TestExecution_ExecuteOnDestContext_Wrong(t *testing.T) {
 
 	vmOutput, err := host.RunSmartContractCall(input)
 	require.Nil(t, err)
-	expectedVMOutput := expectedVMOutput_DestCtx_WrongContractCalled(parentCode)
-	require.Equal(t, expectedVMOutput, vmOutput)
+	require.NotNil(t, vmOutput)
+
+	if host.Runtime().ElrondSyncExecAPIErrorShouldFailExecution() == false {
```

Comment:
weird having == false, maybe make it !host..... or invert the conditions

Reply:
this is only to check that the code was there.